src/lab4.py

```diff
@@ -0,0 +1,52 @@
+class Insect:
+#конструктор
```

Comment:
It is necessary to improve the readability of the code, namely comments such as #constructor, #destructor, etc. need to be separated from the code to add health and bottom samples